include/config.h

```diff
@@ -8,10 +8,16 @@
 
 #include <sel4/config.h>
 
-/* Set ENABLE_SMP_SUPPORT for kernel source files */
 #ifdef CONFIG_ENABLE_SMP_SUPPORT
```

Comment:
I don't see the benefit of this commit?

Reply:
The SMP conditional macros are use all over the code and thus seem generic, defining them in `statedata.h` seems the wrong place.

Reply:
I assume you comment refer to the commit https://github.com/seL4/seL4/pull/752/commits/37a0d3b5d34b0c4eaf7e0e836f64b9e5c3302020 (define SMP conditional macros in config.h). The SMP conditional macro are used in various place in the code, thus defining them here seems better place than `statedata.h`. Using `util.h` is another option, but thy seem quite closely related to the global configuration.